src/notifier.js

```diff
@@ -28,6 +28,7 @@ class Notifier {
     this.queueUrl = this.sqs.createQueue({
       QueueName:  this.options.queueName,
     }).promise().then(req => req.QueueUrl);
+    this.sendTimes = {};
   }
```

Comment:
You should probably do a setInterval to periodically purge old entries where all timestamps are past 10min.. so we don't run out of memory, particularly in a world where it's not restarted every 24 hours.

Alternatively, instead of using address as a key, you can use `hash(address + salt) % 64000` there will obviously be some collisions, but 64000 will mean there is few. The salt would be randomly generated at startup, every startup, and it protects from an attacker using your rate limit to silence alerts...